api/src/controllers/mission.js

```diff
@@ -66,7 +66,16 @@ router.post("/", passport.authenticate("referent", { session: false }), async (r
   try {
     const { error, value: checkedMission } = validateMission(req.body);
     if (error) return res.status(400).send({ ok: false, code: ERRORS.INVALID_BODY, error });
+
     const data = await MissionObject.create(checkedMission);
+
+    const referentDepartment = await UserObject.findOne().and([{ role: "referent_department" }, { department: checkedMission.department }]);
```

Comment:
pourquoi pas :
```suggestion
    const referentDepartment = await UserObject.findOne({ role: "referent_department", department: checkedMission.department });
```
?

de plus, on a des constantes qu'on préfère utiliser -> `ROLES` .

Je te laisse fouiller dans le code pour voir comment elles sont importées/utilisées